config/locales/nl.yml

```diff
@@ -35,7 +35,7 @@ nl:
         description_nl: Nederlandse beschrijving
         end_date: Einddatum
         end_time: Eindtijd
-        google_event: Voeg toe aan je agenda
+        google_event: Kopiëer eenmalig naar kalender
```

Comment:
Feel like the instruction of only copying once should be more elaborate and not incorporated into a title. Pretty sure people will not understand this.

Reply:
Not sure how to fix this. There's only limited space, and as it's a hyperlink it would be weird to just add a button next to it for more info. I guess I will add one of those little question marks in a circle

Reply:
Now that I think of it, I don't think that adding more information might be necessary.

The whole thing I am trying to convey, is that when you add the activity to your calendar through that hyperlink, that it does not 'subscribe' your calendar to the activity, but simply copies it over as-is.

When the text says 'copy once to calendar, and the opened webpage displays a filled in template for that single activity, my hopes are that the user will understand that he should not rely on that activity in his calendar to be updated when koala updates.

What do you think?

Reply:
I believe things are clear, now that I give an explanation on the fact that koala is only aware of the activities in koala